Bio/ExPASy/ScanProsite.py

```diff
@@ -28,21 +28,21 @@ def __init__(self):
         self.capped = None
         self.warning = None
 
-
-def scan(seq="", mirror="https://www.expasy.org", output="xml", **keywords):
+### October 28th 2020 it was recognised that the main url of prosite changed from https://www.expasy.org to https://prosite.expasy.org. Thus a change in the mirror was issued from https://www.expasy.org to https://prosite.expasy.org.
```

Comment:
I'm not sure if we are enforcing a line-length limit now (I guess not) but would be nice to break this into two lines

Reply:
Do you know when the change occurred? Not necessarily important but more informative than the date of recognition.

Reply:
No, I can only say that it worked October 10th and then not yesterday.

Reply:
That's OK then. It seems that at some time between 2008 and 2012 they changed it...(according to their papers)

Reply:
...but obviously they turned-off the old URL just now.